ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -130,6 +130,13 @@ if should_run_integration_tests; then
   )
 
   readonly EMULATOR_SCRIPT="run_integration_tests_emulator_bazel.sh"
+
+  echo
+  io::log_yellow "running storage integration tests via Bazel+Emulator"
+  echo
```

Comment:
Perhaps this should match lines 148-149 below, and just do a single "==="s banner and no extra blank lines?

Reply:
Done.